cmd/fortifyExecuteScan.go

```diff
@@ -65,6 +66,8 @@ func fortifyExecuteScan(config fortifyExecuteScanOptions, telemetryData *telemet
 	if err != nil {
 		log.Entry().WithError(err).Fatal("Fortify scan and check failed")
 	}
+	// make sure that no specific error category is set in success case
+	log.SetErrorCategory(log.ErrorUndefined)
```

Comment:
The comment hints that this is apparently necessary... but my gut feeling is that something is wrong, if that is the case. I am not a fan of how the error categorization needs to be used. It feels like an orthogonal layer that adds information that should instead come from the errors themselves. We could have our own error package that produces or wraps errors into errors of certain types, and the category follows from there.